### ISSUE TITLE
Remove user profile

### ISSUE THREAD

**Issue by wsilva** [ricardovaleriano/call4paperz#108]
Let the user remove its own profile

**Comment by lucianosousa** [comment]
@tauil I like this. Looks respectful

**Comment by tauil** [comment]
Agreed :+1:

**Comment by lucianosousa** [comment]
@wsilva @tauil 

Question: What do you guys think we need to do with:

- Created events;
- Proposals;
- Comments;
- Votes;

I can imagine a lot of strange situations here:

- Few events will just disappear if it's gone with the users. Not cool. My suggestion is do not remove it.
- Few proposals will also disappear, not cool. My suggestion is remove proposals just from open events. If the proposal belongs to an old event, we put the name or other required info in the proposal, but keep them in the system.
- Comments: keep all of them, if the user delete their profile, we put the User Name on it.
- Votes: remove from open events and keep them from closed ones.

----

We can also create an kind of user named `Removed User` to keep all of the infos from old users on it

Keep in mind that bugs with votes can occur on this issue. Also, we can lost a lot of useful infos, such as proposals and events.

After we decide what to do about it I can move forward with this issue.

**Comment by tauil** [comment]
I agree that we may lose a lot of useful info, but if the info belongs to a user that wants to remove his profile, it's fair to at least ask him if he wants to delete everything he inputted or just his profile. That way we would have two 'Remove profile options'. Just remote profile and all info would have a 'Removed user' in the author or Remove everything that the user created. We would not remove the data from the DB, just do a logical removal with a field like `deleted: true` and then replace the content by a message that the content has been removed from the author. Those contents could not be voted anymore.
